### PR TITLE
GA-ISSUE 242: [Golden Age] - Build Confirmation Page

### PR DESCRIPTION
**Pull request by MichaelKostroma**

### CODE REVIEW

File: ui/pages/Settings.tsx [hunk 1]
```diff
@@ -109,15 +108,15 @@ function SettingRow(props: {
...
+            padding-top: 12px;
             display: flex;
             justify-content: space-between;
             align-items: center;
 
             color: var(--green-20);
-            font-size: 18px;
-            font-weight: 600;
-            line-height: 24px;
+            font-size: 14px;
```
Comment (rileystephens28) [inline ui/pages/Settings.tsx]: Change `font-size` to `16px`

File: ui/pages/Settings/SettingButton.tsx [hunk 2]
```diff
@@ -24,9 +24,9 @@ export default function SettingButton(props: {
         <style jsx>{`
           .action_name {
             color: var(--green-20);
-            font-size: 18px;
-            font-weight: 600;
-            line-height: 24px;
+            font-size: 14px;
```
Comment (rileystephens28) [inline ui/pages/Settings/SettingButton.tsx]: Change `font-size` to `16px`

**Review by rileystephens28 — COMMENTED** [review]
There is a lot of code commented out in `ui/pages/Settings.tsx` that needs to be removed for the audit.

File: ui/pages/Settings.tsx (hunk 1 above)
Comment (MichaelKostroma) [reply ui/pages/Settings.tsx]: done ✅

File: ui/pages/Settings/SettingButton.tsx (hunk 2 above)
Comment (MichaelKostroma) [reply ui/pages/Settings/SettingButton.tsx]: done ✅

**Review by rileystephens28 — APPROVED** [review]

**Pull request merged**